src/kiwixchoicebox.cpp

```diff
@@ -186,11 +182,8 @@ bool KiwixChoiceBox::addSelection(QListWidgetItem *item, bool updateRequired)
 
 bool KiwixChoiceBox::removeSelection(QListWidgetItem *item)
 {
-    auto selection = item->text();
-    if (!m_choiceItems.contains(selection)) return false;
```

Comment:
Shouldn't we handle the case when `ui->currentChoices` doesn't contain `item->text()` ?